remoulade/api/schema.py

```diff
@@ -7,7 +7,7 @@ class MessageSchema(Schema):
     """
 
     actor_name = fields.Str(validate=validate.Length(min=1), required=True)
-    args = fields.List(fields.Raw())
-    kwargs = fields.Dict()
-    options = fields.Dict()
-    delay = fields.Number(validate=validate.Range(min=1))
+    args = fields.List(fields.Raw(), allow_none=True)
+    kwargs = fields.Dict(allow_none=True)
+    options = fields.Dict(allow_none=True)
```

Comment:
after some thougth, in send_with_options options is used as kwargs. so it need to be a dict.
Also it's used as a parameter.

You should enhance you test `test_enqueue_message`.
`broker.enqueue` should be mocked instead of `actor.send_with_options` and you should test a real option like `time_limit`.